src/components/list/detail.css

```diff
@@ -1,18 +1,14 @@
-@value colorMustard, colorBlue, textFontSizeM, textLineHeightM, textFontSizeL, textLineHeightL, sizeXxs, sizeS, sizeL from '../../utils/basics.css';
+@value colorMustard, colorBlue, textFontSizeM, textLineHeightM, textFontSizeL, textLineHeightL, sizeXxs, sizeS, sizeL, sizeXl from '../../utils/basics.css';
 
 .flipIconContainer {
   width: sizeL;
   height: sizeL;
   border-radius: 50%;
-  position: fixed;
+  position: absolute;
```

Comment:
we do not need desktop part anymore, with absolute it's positioned properly according to the parent